corehq/apps/app_manager/add_ons.py

```diff
@@ -150,6 +155,9 @@ def _grandfathered(slug, app):
         "created them. Available in form settings."),
         help_link="https://confluence.dimagi.com/display/commcarepublic/Child+Cases",
         used_in_form=lambda f: f.form_type != "module_form" or bool(f.actions.subcases),
+        privilege=CHILD_CASES,
+        upgrade_text=_("Child cases are not available on your subscription. "
```

Comment:
Non-blocking: if this text were more generic, like, "This feature is not available..." could you get rid of this parameter altogether?